Added latest update

ManicZanic opened the pull request:
Added REV23 and removed REV22

ManicZanic committed:
Added latest update

Added REV23 and removed REV22

ManicZanic committed:
Add topsy and turvey combined pack n go

ManicZanic committed:
Added topsy turvey combined pack n go

Follansbeast commented:
This removes all edits associated with rev 22. Creating new versions and deleting old versions eliminates ability to diff and perform code reviews. This should be resubmitted using methods appropriate to git.

Perrsonally would elect to get away from "rev ##" completely. If these are actually different, they should be separate forks or feature branches, but not have the files renamed and moved.

ManicZanic committed:
Update file system to properly use GIT as intended

Reuploaded REV22 module and moved all old module files into the new Pre-Training Archive folder. Still need to incorporate the REV22 code into the latest module files.
